tsconfig.json

```diff
@@ -2,9 +2,9 @@
 	"extends": "@sindresorhus/tsconfig",
 	"compilerOptions": {
 		"outDir": "dist",
-		"target": "es2018",
+		"target": "es2020",
```

Comment:
Change because test failed.

```zsh
Uncaught exception in test/bigint.ts
 
   test/bigint.ts:12:6 - error TS2737: BigInt literals are not available when targeting lower than ES2020.
 
   12    ow(9007199254740991n, ow.bigint);
            ~~~~~~~~~~~~~~~~~
```